Index.html

```diff
@@ -11,318 +11,355 @@
-						sign-ups required. has been the industry's standard dummy text ever
-						since the 1500s, when an unknown printer took a standard dummy text.
-					</p>
-					<div class="programmingLanguages">
-						<ul class="languagesUsed">
-							<li>css</li>
-							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
-							<li>html</li>
-							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
-							<li>bootstrap</li>
-							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
-							<li>Ruby</li>
-						</ul>
+					<div class="project4Info">
+						<h3 class="multiPost">Multi-Post Stories</h3>
+						<p class="projectBrief">
+							A daily selection of privately personalized reads; no accounts or
+							sign-ups required. has been the industry's standard dummy text ever
+							since the 1500s, when an unknown printer took a standard dummy text.
+						</p>
+						<div class="programmingLanguages">
+							<ul class="languagesUsed">
+								<li>css</li>
+								<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+								<li>html</li>
+								<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+								<li>bootstrap</li>
+								<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+								<li>Ruby</li>
+							</ul>
+						</div>
+						<button class="projectButton" type="Button">See Project</button>
 					</div>
-					<button class="projectButton" type="Button">See Project</button>
 				</div>
 			</div>
-			<img class="line" src="./assets/vectorLine.svg" alt="horizontal line" />
 		</section>
 		<footer class="contact">
-			<div class="contact-text">
-				<h2>CONTACT US</h2>
-				<p>
-					If you have an appication you are interested in developing, a feature
-					that you need built or a pooject that needs coding, i'd love to help
-					with it
-				</p>
+			<div class="contactForm">
+				<div class="contact-text">
+					<h2 class="contactUsHeading">CONTACT US</h2>
+					<h2 class="chat">Let's Chat!</h2>
+					<p>
+						If you have an appication you are interested in developing, a feature
+						that you need built or a pooject that needs coding, i'd love to help
+						with it
+					</p>
+				</div>
+				<form action="https://formspree.io/f/meqnjrwb" method="post">
+					<input
+						name="name"
+						id="name"
+						class="formEl"
+						type="text"
+						placeholder="Your name"
+						maxlength="30"
+						required
+					/>
+					<input
+						name="email"
+						id="email"
+						class="formEl"
+						type="email"
+						placeholder="Your email address"
+						required
+					/>
+					<textarea
+						name="message"
+						id="message"
+						class="formEl"
+						cols="30"
+						rows="10"
+						maxlength="500"
+						placeholder="Please leave your message here"
+						required
+					></textarea>
+					<input class="projectButton btn" type="submit" value="See Project" />
```

Comment:
- [ ] [OPTIONAL] I would suggest updating the value of the button to "Send" instead of "See Project" according to the Figma design. I will mark this as [OPTIONAL] because it's a minor change but I strongly suggest updating this value :smiley: :raised_hands: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144443638-0a479887-1789-4df4-b82e-d3053688a257.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144443679-5283f9e7-b79f-4278-9ee7-34f6a16a00b5.png)